src/culc/TestCulc.java

```diff
@@ -0,0 +1,138 @@
+                arabic -= 10;
+
+            } else if (arabic % 100 == 20) {
+                romanNumbersOut[1] = romanNumber[10];
+                arabic -= 20;
+
+            } else if (arabic % 100 == 30) {
+                romanNumbersOut[1] = romanNumber[11];
+                arabic -= 30;
+
+            } else if (arabic % 100 == 40) {
+                romanNumbersOut[1] = romanNumber[12];
+                arabic -= 40;
+
+            } else if (arabic % 100 == 50) {
+                romanNumbersOut[1] = romanNumber[13];
+                arabic -= 50;
+
+            } else if (arabic % 100 == 60) {
+                romanNumbersOut[1] = romanNumber[14];
+                arabic -= 60;
+
+            } else if (arabic % 100 == 70) {
+                romanNumbersOut[1] = romanNumber[15];
+                arabic -= 70;
+
+            } else if (arabic % 100 == 80) {
+                romanNumbersOut[1] = romanNumber[16];
+                arabic -= 80;
+            }else if (arabic % 100 == 90) {
+                romanNumbersOut[1] = romanNumber[17];
+                arabic -= 90;
+            }else if (arabic % 1000 == 100) {
+                romanNumbersOut[2] = romanNumber[18];
+                arabic -= 100;
+            }else if (arabic % 1000 == 200) {
+                romanNumbersOut[2] = romanNumber[19];
+                arabic -= 200;
+            }else if (arabic % 1000 == 300) {
+                romanNumbersOut[2] = romanNumber[20];
+                arabic -= 300;
+            }else if (arabic % 1000 == 400) {
+                romanNumbersOut[2] = romanNumber[21];
+                arabic -= 400;
+            }else if (arabic % 1000 == 500) {
+                romanNumbersOut[2] = romanNumber[22];
+                arabic -= 500;
+            }else if (arabic % 1000 == 600) {
+                romanNumbersOut[2] = romanNumber[23];
+                arabic -= 600;
+            }else if (arabic % 1000 == 700) {
+                romanNumbersOut[2] = romanNumber[24];
+                arabic -= 700;
+            }else if (arabic % 1000 == 800) {
+                romanNumbersOut[2] = romanNumber[25];
+                arabic -= 800;
+            }else if (arabic % 1000 == 900) {
+                romanNumbersOut[2] = romanNumber[26];
+                arabic -= 900;
+            }else if (arabic % 10000 == 1000) {
+                romanNumbersOut[3] = romanNumber[27];
+                arabic -= 1000;
+            }
+        }
+        if (romanNumbersOut[5] != null)
+            System.out.print(romanNumbersOut[5]);
+        if (romanNumbersOut[4] != null)
+            System.out.print(romanNumbersOut[4]);
+        if (romanNumbersOut[3] != null)
+            System.out.print(romanNumbersOut[3]);
+        if (romanNumbersOut[2] != null)
+            System.out.print(romanNumbersOut[2]);
+        if (romanNumbersOut[1] != null)
+            System.out.print(romanNumbersOut[1]);
+        if (romanNumbersOut[0] != null)
```

Comment:
This logic and above blocks you can wrap Into cycle and then:
romanNumbersOut[i]!=null